src/page-objects/storefront/AccountOrder.ts

```diff
@@ -8,7 +8,7 @@ export class AccountOrder implements PageObject {
     public readonly digitalProductDownloadButton: Locator;
 
     constructor(public readonly page: Page) {
-        this.orderExpandButton = page.getByRole('button', {name: 'Expand'}).first();
+        this.orderExpandButton = page.locator('.order-hide-btn').first();
```

Comment:
This is from @tobiasberge 's PR here: #126 

I rebased on top of that, because the tests were failing without it.

Reply:
https://github.com/shopware/acceptance-test-suite/pull/126/files#r1766818473

Reply:
This was failing because the translation was changed from "Expand" to "Show details". And when just changing the name I guess it would stop working in older SW versions that don't have the updated translation.

Reply:
We can merge the other PR first